pulp_ansible/app/viewsets.py

```diff
@@ -70,6 +73,38 @@ class CollectionFilter(ContentFilter):
     FilterSet for Ansible Collections.
     """
 
+    latest = BooleanFilter(field_name='latest', method='filter_latest')
+
+    def filter_latest(self, queryset, name, value):
+        """
+        If the value of 'latest' is True, exclude all non-latest Collections from the results.
+
+        Args:
+            queryset: The already-formed queryset for modification
+            name: The name of the parameter, 'latest'
+            value: The value of the argument. This is checked if 'True' or not.
+
+        Returns:
+            Queryset with non-latest collections excluded if value is True.
+        """
+        if value:
```

Comment:
This is nitpick but a guard clause might be better here. 

https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html

Reply:
I agree but this if statement isn't nested though?

Reply:
So like with a guard clause, this would be:

```python
if not value:
    return queryset

namespace_name_dict = defaultdict(lambda: defaultdict(list))
# and so on...
```

Reply:
Oh word that is way better. Thank you for explaining this to me. I'm pushing the fix now.